lib/src/block_syntaxes/table_syntax.dart

```diff
@@ -50,7 +51,7 @@ class TableSyntax extends BlockSyntax {
       if (children != null) {
         while (children.length < columnCount) {
           // Insert synthetic empty cells.
-          children.add(Element.empty('td'));
+          children.add(Element('td', const []));
```

Comment:
Should change to:
```dart
children.add(Element('td', []));
```

Reply:
It's a bit of a hack to support a hack, but yes.

Once the flutter markdown integration test is added, it'll be clear when this causes failures.